msi/GitProduct.wxs

```diff
@@ -28,13 +28,99 @@
     <!-- Other MSI specific properties -->
     <Property Id='MSIFASTINSTALL' Value='7' />
 
+    <!-- Git for Windows MSI Properties -->
+    <!-- CLIENVIRONMENT Property implementing the "Remember Property" pattern -->
+    <Property Id="CLIENVIRONMENT" Value="Bash">
+      <RegistrySearch Id="CliEnvironmentSearch" Root="HKLM" Key="SOFTWARE\GitForWindows" Name="CliEnvironment" Type="raw" />
+    </Property>
+    <SetProperty Id="CMDLINE_CLIENVIRONMENT" Value="[CLIENVIRONMENT]" Before="AppSearch" Sequence="first" />
+    <SetProperty Id="CLIENVIRONMENT" Value="[CMDLINE_CLIENVIRONMENT]" After="AppSearch" Sequence="first">CMDLINE_CLIENVIRONMENT</SetProperty>
```

Comment:
Okay. I would have preferred to see those properties to be introduced together with the actual options, but I won't insist.